src/utils/validations.ts

```diff
@@ -335,3 +335,9 @@ export function isAddress(address: string) {
   }
   return true;
 }
+
+export const isFnSelector = (h: string): boolean => {
+  if (h.length !== 8) return false;
+  var a = parseInt(h, 16);
```

Comment:
Why Var instead of const or let?

Also, can the variables be a bit more descriptive?
I'm having a hard time reading/understanding this at a glance